tests/api/utils/test_graph.py

```diff
@@ -0,0 +1,55 @@
+from __future__ import annotations
+
+import pytest
+from snowflake.cli.api.utils.graph import Graph, Node
```

Comment:
hmm, what's the coverage of the graph code here? Looks like you missed a few parts of the contract.

Reply:
added more coverage.